CYF-London| Zohreh Kazemianpour| Wireframe task | Week1

zohrehKazemianpour opened the pull request:
<!--

You must title your PR like this:

COHORT_NAME | FIRST_NAME LAST_NAME | REPO_NAME | WEEK

For example,

CYF-London| Zohreh Kazemianpour| Wireframe task | Week1

Complete the task list below this message.
If your PR is rejected, check the task list.

-->

## Learners, PR Template

Self checklist

- [x] I have committed my files one by one, on purpose, and for a reason
- [x] I have titled my PR with COHORT_NAME | FIRST_NAME LAST_NAME | REPO_NAME | WEEK 
- [x] I have tested my changes
- [x] My changes follow the [style guide](https://syllabus.codeyourfuture.io/guides/code-style-guide/)
- [x] My changes meet the [requirements](./README.md) of this task

## Changelist

 -Added Google Fonts for better typography.
- Created a hero section with an introduction to Git.
- Designed two card articles explaining the features and branches of Git.
- Included fixed footer with a link to personal portfolio website.

## Questions

I've used Flexbox for the card layout. Should I consider using CSS Grid for more flexibility, or does Flexbox handle this layout well across different screen sizes?

zohrehKazemianpour committed:
HTML initial draft

zohrehKazemianpour committed:
styled the main class in CSS

zohrehKazemianpour committed:
styledheader

zohrehKazemianpour committed:
styling the buttons

zohrehKazemianpour committed:
fixing footer at the bottom of the veiwpoint

zohrehKazemianpour committed:
adding flexbox to .card

zohrehKazemianpour committed:
adding first child pseudo class

zohrehKazemianpour committed:
adding@media quries

zohrehKazemianpour committed:
updated the colour of Header

cjyuan commented:
The page header includes a title but not a description (In the wireframe, there is a short description below the title)
 Your :first-child selector is currently applied to the 2nd article. Also, the style you used is not quite standing out. (Initially, I just assumed the scroll bar was not showing up in the 3rd article because its summary is not long enough.)

Besides these minor issues, your page looks good.

zohrehKazemianpour committed:
Added a brief description bellow the title

zohrehKazemianpour committed:
updated  :first-child selector

zohrehKazemianpour commented:
@cjyuan  Thank you for the feedback! I really appreciate it. I have incorporated the changes you requested:
Added a brief description to the title.
Applied the :first-child selector to the first article.
Initially, I considered using a different colour for the first article to make it stand out but hesitated because I was concerned it might disrupt the cohesive design of the page. Would you recommend applying a different style to the first article to highlight it more, or would you prefer the more subtle approach?

Additionally, if further refactoring of the HTML is needed to better align with the requirements, I’m happy to make those adjustments.

cjyuan commented:
Page looks better now and meet the acceptance criteria. Just a few suggestions,

1. There is at least one more semantic elements you can use in your page (See: https://developer.mozilla.org/en-US/docs/Glossary/Semantics)

2. On wide screen, the buttons can go beyond the boundary of `<article>` (the white rounded rectangular regions).

3. The last two articles can appear beneath the footer (their buttons could be partially blocked by the footer). You can try enclose the articles in a "container" element, and then make sure that the container element will never overlap the footer.

zohrehKazemianpour committed:
fixed the button issue

zohrehKazemianpour committed:
added more semantic element

zohrehKazemianpour commented:
@cjyuan Thanks for the input. I have corrected the issues you mentioned.
